src/cluster.c

```diff
@@ -1718,6 +2054,10 @@ void readwriteCommand(client *c) {
     addReply(c,shared.ok);
 }
 
+void clusterCommonInit(void) {
```

Comment:
It feels a bit redundant, we can call clusterAsmInit directly, do i miss something?